Client/Frontend/UserContent/UserScripts/MainFrame/AtDocumentEnd/MetadataHelper.js

```diff
@@ -5,11 +5,44 @@
 
 "use strict";
 
-const metadataparser = require("page-metadata-parser/parser.js");
+const {getMetadata:metadataparser, metadataRuleSets} = require("page-metadata-parser");
 
 function MetadataWrapper() {
   this.getMetadata = function() {
-    return metadataparser.getMetadata(window.document, document.URL);
+    const customRuleSets = metadataRuleSets;
```

Comment:
Is this code yours or taken from somewhere?

Reply:
This whole file uses https://github.com/mozilla/page-metadata-parser